src/Displaz.jl

```diff
@@ -6,6 +6,8 @@ using Colors
 export plot3d, plot3d!, clearplot, viewplot
 export KeyEvent, CursorPosition, event_loop
 
+displaz_cmd() = get(ENV, "DISPLAZ_CMD", "displaz")
```

Comment:
I'd prefer to do this once at package initialization (or possibly build time?).  Is there a reason it should happen at every call?

Also the call to displaz should be changed in events.jl as well.